### PR TITLE
RangeError when useGrouping is string 'undefined'

### PR DESCRIPTION
**Pull request by FrankYFTang**
per https://github.com/tc39/proposal-intl-numberformat-v3/pull/107

### CODE REVIEW

**Comment by FrankYFTang** [comment]
@romulocintra @sffc @ryzokuken

File: test/intl402/NumberFormat/test-option-useGrouping-extended.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ assert.sameValue(render({notation: 'compact', useGrouping: false}), false, 'comp
 assert.sameValue(render({notation: 'compact', useGrouping: null}), false, 'compact, null');
 assert.sameValue(render({notation: 'compact', useGrouping: 'min2'}), 'min2', 'compact, "min2"');
 
-assert.sameValue(render({useGrouping: 'undefined'}), 'auto', 'use fallback value');
+assert.throws(RangeError, function () {
```
Comment (romulocintra) [inline test/intl402/NumberFormat/test-option-useGrouping-extended.js:38]: Thanks for catching this @FrankYFTang  I would rather suggest deleting this assertion and add "undefined" string at https://github.com/tc39/test262/blob/main/test/intl402/NumberFormat/test-option-useGrouping.js#L34

**Comment by ptomato** [comment]
Do I understand correctly that we should merge either #3660 or this PR, but not both?

**Comment by FrankYFTang** [comment]
Close this PR and yield to  #3660

**Pull request closed without merging**